cutekit/project.py

```diff
@@ -0,0 +1,106 @@
+from . import cli, model, vt100, shell
+import dataclasses, os
+from pathlib import Path
+from typing import Optional
+from enum import StrEnum
+
+class Suffix(StrEnum):
+    JSON="json"
+    TOML="toml"
+
+class InitArgs:
+    name: str = cli.operand("name", "Set the name of your project")
+    description: str = cli.arg(None,"desc", "Set the description of your project", None)
+    kind: model.Kind = cli.arg(None, "kind", "Kind of the manifest", model.Kind.PROJECT)
+    suffix: Suffix = cli.arg(None, "prefer", "Manifest format to use", Suffix.JSON)
+
+@cli.command("init", "Initialize your manifest.")
+def init_manifest(args: InitArgs):
+    if not vt100.ask(f"'{vt100.rgb(235, 111, 146) + args.name + vt100.RESET}' is the name of your project  ?"):
```

Comment:
You should avoid using `+` when formatting
```suggestion
    if not vt100.ask(f"'{vt100.rgb(235, 111, 146)} {args.name}{vt100.RESET}' is the name of your project ?"):
```

When asking for a `--kind=exe` it will also be prompted (!= then a project)

Also is this prompt necessary ?